css/global.css

```diff
@@ -3,6 +3,14 @@
   display: none !important;
 }
 
+/* hides "over 100 applicants" message at linkedin.com/jobs/collections */
+
+.job-details-jobs-unified-top-card__primary-description-container:has(div):has(
```

Comment:
This looks for an element with the class `job-details-jobs-unified-top-card__primary-description-container` that has a child div that has a child element with the class `tvm__text--neutral`, then removes said element from the DOM.